src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.figures;
+
+public abstract class Figure implements Drawable {
+    protected String color;
+
+    protected Figure(String color) {
+        this.color = color;
+    }
+
+    protected abstract double getArea();

```

Comment:
because I have no idea in what context the AreaCalculator can be used except for shapes, and I thought that it can be implemented as an abstract method of shapes.

> Why you placed this method in class not in the interface?

Reply:
let's create another interface called AreaCalculator and transfer this method there

Reply:
> let's create another interface called AreaCalculator and transfer this method there

I've done it, but cann't understand. Why we need this like interface. I asked about it on QnA, and the answer was something like: "If I needed to do this, I would make getArea as an abstract method". Is it something like: try to do most of abstract methods in intefaces or no?

Reply:
@Andriy-dz  @liliia-ponomarenko in my opinion there is no problem in placing `getArea()` inside of the abstract `Figure` class

let's discuss internally if you disagree :)